package.json

```diff
@@ -4,9 +4,9 @@
   "description": "Web of Things gateway",
   "main": "app.js",
   "scripts": {
-    "test": "./test/run-tests.sh",
+    "test": "npm run lint && npm run mocha",
     "lint": "./test/lint-all.sh",
-    "mocha": "mocha"
+    "mocha": "./test/run-tests.sh"
```

Comment:
So perhaps we should have a mocha-all.sh and run-tests.sh would just call lint-all.sh and mocha-all.sh